R/sql_methods.R

```diff
@@ -0,0 +1,108 @@
+source("R/get_connection.R")
+require(reticulate)
+
+# installing python-based Nuvolos connector, if not installed yet
+tryCatch({
+  import("nuvolos")
+}, error = function(e) {
+  if (is_local()){
+    py_install("nuvolos", pip =TRUE)
+  } else {
+    py_install("nuvolos-odbc", pip =TRUE)
+  }
+})
+
+
```

Comment:
@pibence is this code required to be run at package load time? I would prefer it to be removed and used only when first calling the read_sql